src/services/API/Database/index.ts

```diff
@@ -0,0 +1,58 @@
+import auth from '@react-native-firebase/auth';
+import {firebase, FirebaseDatabaseTypes} from '@react-native-firebase/database';
+import {nanoid} from 'nanoid';
+
+export interface UserInfo {
+  code: string;
+  invited: number;
+}
+
+const db = firebase
+  .app()
+  .database(
+    'https://referralapp-beb46-default-rtdb.europe-west1.firebasedatabase.app/',
+  );
+
+enum DatabaseEndpoints {
+  users = 'users/',
+}
+
+class DatabaseAPI {
+  static incrementRefValue = async (refCode: string) => {
+    const res = await db.ref(DatabaseEndpoints.users).once('value');
+    for (const [key, {code, invited}] of Object.entries<UserInfo>(res.val())) {
+      if (code === refCode) {
+        await db
+          .ref(`${DatabaseEndpoints.users}${key}`)
+          .update({invited: invited + 1});
+      }
+    }
```

Comment:
hmmmmm, so there is a possibility that multiple users are going to have same refCodes? I don't think so, better to not use loop here.